### ISSUE TITLE
Documentation of dts stat is lacking

### ISSUE THREAD

**Issue by cdowd** [cdowd/twosamples#1]
Without anything formal to cite, this needs substantially better documentation

**Comment by cdowd** [comment]
The documentation for this needs to be improved both here/in general, but also specifically the readme here needs to be made available on the manual pages.

**Comment by koo-ec** [comment]
Thanks for your fantastic R package. 
It should be mentioned what does DTS stands for?

**Comment by cdowd** [comment]
Glad you like it. I appreciate feedback. 

I don't have a great answer, you aren't the first to ask. It doesn't stand for anything really. Initially, the code was for the KS test -- and the notation in their paper denotes the test statistic as D, so when I coded it up, I used dts -- for D test stat (one letter function names always make me skittish). When I decided to improve on it, I kept that name (copying functions into new files, etc)-- but then when I decided to build a package, everything which could have a real name (KS, kuiper, etc) was given its proper name, and dts didn't have one, so it was left alone. About the same time, everything got split out into stat and test functions -- leaving me with dts_stat and dts_test functions. I didn't notice until after package publication that dts_stat & dts_test are meaningless and redundant gibberish.  

"D test stat" is the truest answer, but it won't help anybody trying to google. "Dowd test stat" (my name) would be the answer of an egomaniac -- and require an order of magnitude more confidence that this was a novel contribution.  "DTS test stat" would be the classic programmer answer -- with a recursive acronym, and it would actually maybe be productive for google, but it feels absurd. So for now, my position is that it is gibberish and maybe I will learn to use better names in my code going forwards. 

Unfortunately, for backwards compatibility reasons -- when I update the package, the name will have to stay -- I wouldn't want to break someone's code because I know it would make me furious.  If you (or anybody) has suggestions for what the *official* acronym could be, I'm all ears.
